docs/siem/detections/api/lists/api-find-list-containers.asciidoc

```diff
@@ -0,0 +1,94 @@
+[[lists-api-find-list-containers]]
+=== Find list containers
+
+Retrieves a paginated subset of list containers. By default, the first page is
+returned with 20 results per page.
+
+==== Request URL
+
+`GET <kibana host>:<port>/api/lists/_find`
+
+===== URL query parameters
+
+All parameters are optional:
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description
+
+|`page` |Integer |The page number to return.
+
+|`per_page` |Integer |The number of list containers to return per page.
+
+|`sort_field` |String |Determines which field is used to sort the results.
+
+|`sort_order` |String |Determines the sort order, which can be `desc` or `asc`.
+
+|`cursor` |String |Returns the containers that come after the last container
+returned in the previous call (use the `cursor` value returned in the previous
+call). This parameter uses the `tie_breaker_id` field to ensure all containers
+are sorted and returned correctly.
+See {ref}/search-request-body.html#request-body-search-search-after[Search After] for more information.
+
+|`filter` |String a|Filters the returned results according to the value of the 
+specified field, using the `<field name>:<field value>` syntax, where
+`<field name>` can be:
+
+* `name`
+* `type`
+* `created_by`
+* `updated_by`
+
+|==============================================
+
+===== Example request
+
+Retrieves the first two `keyword` list containers, sorted by `name` in
+descending order:
+
+[source,console]
+--------------------------------------------------
+GET api/lists/_find?filter=type:keyword&page=1&per_page=2&sort_field=name&sort_order=desc
+--------------------------------------------------
+// KIBANA
+
+==== Response code
+
+`200`:: 
+    Indicates a successful call.
+    
+==== Response payload
+
+[source,json]
+--------------------------------------------------
+{
+  "cursor": "WzIsWyJFeGNsdWRlIGhvc3RzIiwiMTExZWExZWMtMjIzZC00YjVjLWJjYWYtM2U5NjBkNmMyMzI1Il1d",
+  "data": [
+    {
+      "id": "source.geo.location",
+      "created_at": "2020-07-07T06:13:14.321Z",
+      "created_by": "Threat Hunter",
+      "description": "Contains list items that exclude outgoing traffic from specific locations.",
+      "name": "Exclude outgoing traffic",
+      "tie_breaker_id": "608095d8-dcd4-4b83-9a4a-429831a7b1f9",
+      "type": "keyword",
+      "updated_at": "2020-07-07T06:13:14.321Z",
+      "updated_by": "Threat Hunter"
+    },
+    {
+      "id": "host.name exception list container",
+      "created_at": "2020-07-07T03:55:10.726Z",
+      "created_by": "Threat Hunter",
+      "description": "Contains list items that exclude host names from detection rules.",
+      "name": "Exclude hosts",
+      "tie_breaker_id": "111ea1ec-223d-4b5c-bcaf-3e960d6c2325",
+      "type": "keyword",
+      "updated_at": "2020-07-07T03:55:10.726Z",
+      "updated_by": "Threat Hunter"
+    }
+  ],
+  "page": 1,
+  "per_page": 2,
+  "total": 3
+}
```

Comment:
These will have the newer fields of `_version, version, immutable` and the same rules apply as before which is they can be used for OCC and the immutable is not being utilized at the moment but in the future it might be for immutable lists and pre-packaged lists.

```ts
   {
      "_version": "Wzg2LDFd",
      "id": "ips.txt",
      "created_at": "2020-08-04T20:35:54.416Z",
      "created_by": "yo",
      "description": "File uploaded from file system of ips.txt",
      "immutable": false,
      "name": "ips.txt",
      "tie_breaker_id": "a9c3a101-d121-45a6-815a-07130e115f47",
      "type": "ip",
      "updated_at": "2020-08-04T20:35:54.416Z",
      "updated_by": "yo",
      "version": 1
    },
```